### ISSUE TITLE
Recognize import paths through re-exports

### ISSUE THREAD

**Issue by hoisel** [soates/Auto-Import#26]
Hi, great job! Could also be possible to find import paths througth re-exports (like barrel files http://tattoocoder.com/angular2-barrels/)

**Comment by soates** [comment]
Yeah absolutely - working on this for the next release - stay tuned :)

**Comment by NoNameProvided** [comment]
Do we have any ETA for this?
